config/Settings.ts

```diff
@@ -122,6 +123,18 @@ export const settings: Array<ISetting> = [
         i18nDescription: 'dialogflow_cx_region_desc',
         required: false,
     },
+    {
+        id: AppSetting.DialogflowDefaultLanguage,
+        public: true,
+        type: SettingType.SELECT,
+        packageValue: 'en',
+        i18nLabel: 'agent_version',
```

Comment:
There is no entry for `agent_version` in localization files. Adding a proper description of this setting will help the admin.